### ISSUE TITLE
Airport Weather Station User Experience

### ISSUE THREAD

**Issue by rvernica** [nhasan/Airports#48]
I want to report an interesting user experience. I was looking to find out the density altitude at KMPI, field elevation 2,254ft. I loaded the _Airport Details_ page and scrolled down to the _Weather Stations_ page. I clicked on the first entry and scrolled down to the _Temperature_ section. I read the _Density altitude_ of 600ft! and concluded that this is the density altitude at KMPI.

There is a big user mistake I did. There was no weather reported for KMPI and the application showed the next available station which is KMCE. I did not notice this distinction and firmly believed I got the right weather. There is a 26nm distance and 2,100ft elevation difference between them!

I understand the usefulness of seeing near-by weather stations, but maybe a warning should be in order if the distance or elevation is over a certain threshold.

**Comment by nhasan** [comment]
The airport detail page clearly shows KMCE as 26NM  SW of the field as all
the other nearby wx stations. Does that work for you?

**Comment by rvernica** [comment]
Yes, I agree, it shows the distance. This issue is more of an *experience* issue where the user, in a rush, might just click on the first weather station and assume that it is at the airport or close to the airport. I'm just suggesting to make it more obvious when the station is at a significant distance or  elevation difference. So, maybe have the distance displayed in red if it is over 10 miles or over 1000 feet elevation difference (10 and 1000 are just examples).

**Comment by nhasan** [comment]
Got it. I will look into this.
